### PR TITLE
Simple helper scripts

### PR DESCRIPTION
**Pull request by bthaile**
created helper scripts to:
1. create open orders
2. fill specific buy/sell orders on market outcome
3. do initial report (for DR or other user)
4. check market creator mailbox
5. list markets and give indication if endDate has passed
6. finalize market
7. set the time by passing in unix timestamp

### CODE REVIEW

File: scripts/helpers/do-initial-report.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        meta: auth,
+        tx: { to: marketID  },
+        _payoutNumerators: payoutNumerators,
+        _invalid: invalid,
+        onSent: function (result) {
+          console.log(chalk.yellow.dim("Sent:"), chalk.yellow(JSON.stringify(result)));
+        },
+        onSuccess: function (result) {
+          console.log(chalk.green.dim("Success:"), chalk.green(JSON.stringify(result)));
+          process.exit(1);
```
Comment (epheph) [inline scripts/helpers/do-initial-report.js]: tiny deal, but exit codes are backwards, usually want to exit 0 on success and exit >0 on error

File: scripts/rep-faucet.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ function faucetInAndMigrate(augur, universe, auth, callback) {
     augur.api.ReputationToken.balanceOf({ _owner: auth.address, tx: { to: reputationToken } }, function (err, reputationTokenBalance) {
       if (err) return callback(err);
       console.log("reputationTokenBalance:", reputationTokenBalance);
-      if (new BigNumber(reputationTokenBalance, 10).gt(new BigNumber("100000", 10))) return callback(null);
```
Comment (epheph) [inline scripts/rep-faucet.js]: Might be more correct to make this new `Error("Already loaded")` but not too big a deal for a one-off script

Comment (bthaile) [reply scripts/rep-faucet.js]: I'll make the change

**Comment by bthaile** [comment]
added a few more scripts, the big one is to create lots of liquidity on a market.

**Pull request merged**